include/kframework/evm.md

```diff
@@ -460,8 +462,8 @@ Here we load the correct number of arguments from the `wordStack` based on the s
 The `CallOp` opcodes all interperet their second argument as an address.
 
 ```k
-    syntax InternalOp ::= CallSixOp Int Int     Int Int Int Int
-                        | CallOp    Int Int Int Int Int Int Int
+    syntax InternalOp ::= CallSixOp Gas Int     Int Int Int Int
+                        | CallOp    Gas Int Int Int Int Int Int
```

Comment:
These should probably remain `Int`? They are being pulled from the `<wordStack>`, which shuold only have `Int`.

Reply:
Okay, I think that might have been legacy from when I had `Gas` permissible on the wordstack